### ISSUE TITLE
React Native components throwing error

### ISSUE THREAD

**Issue by andrew-levy** [LegendApp/legend-state#32]
**Legend State Version: 0.19.7**

When trying to use the react native components from ` @legendapp/state/react-native-components`, I get the following error: 

<img width="362" alt="Screen Shot 2022-10-11 at 9 44 12 PM" src="https://user-images.githubusercontent.com/29075740/195229658-45b534a5-aaa8-4c15-9dd4-d22af9d58ef5.png">

This is the code that caused it. I noticed that the react (web) components entry exports a `legend` object with components that have reactive props, while the react native entry exports `Legend` with components without reactive props. I also can't pass in observables to bindable views like `TextInput` without a typescript and runtime error. Do these components need to be updated to reflect the recent api changes, or am i missing a step to get them working?

```tsx
import { View } from "react-native";
import { observer, useObservable } from "@legendapp/state/react";
import { Legend } from "@legendapp/state/react-native-components";

export default observer(function App() {
  const text = useObservable("Hello World");
  return (
    <View>
      <Legend.Button title="hi" />
      <Legend.TextInput value={text} />
    </View>
  );
});
```

**Comment by jmeistrich** [comment]
I'm reproducing that here. Will get a fix out asap.

**Comment by jmeistrich** [comment]
This should be fixed in 0.19.8. But please let me know if it's still not working right.
